man/sysupdate.d.xml

```diff
@@ -0,0 +1,875 @@
+            <xi:include href="standard-specifiers.xml" xpointer="B"/>
+            <xi:include href="standard-specifiers.xml" xpointer="H"/>
+            <xi:include href="standard-specifiers.xml" xpointer="l"/>
+            <xi:include href="standard-specifiers.xml" xpointer="m"/>
+            <xi:include href="standard-specifiers.xml" xpointer="M"/>
+            <xi:include href="standard-specifiers.xml" xpointer="o"/>
+            <xi:include href="standard-specifiers.xml" xpointer="v"/>
+            <xi:include href="standard-specifiers.xml" xpointer="w"/>
+            <xi:include href="standard-specifiers.xml" xpointer="W"/>
+            <xi:include href="standard-specifiers.xml" xpointer="T"/>
+            <xi:include href="standard-specifiers.xml" xpointer="V"/>
+            <xi:include href="standard-specifiers.xml" xpointer="percent"/>
+          </tbody>
+        </tgroup>
+      </table>
+  </refsect1>
+
+  <refsect1>
+    <title>Examples</title>
+
+    <example>
+      <title>Updates for a Verity Enabled Secure OS</title>
+
+      <para>With the following three files we define a root file system partition, a matching Verity
+      partition, and a unified kernel image to update as one. This example is an extension of the example
+      discussed earlier in this man page.</para>
+
+      <para><programlisting># /usr/lib/sysupdate.d/50-verity.conf
+[Transfer]
+ProtectVersion=%A
+
+[Source]
+Type=http-file
+Path=https://download.example.com/
+MatchPattern=foobarOS_$v_$u.verity.xz
+
+[Target]
+Type=partition
+Path=auto
+MatchPattern=foobarOS_$v_verity
+MatchPartitionType=root-verity
+PartitionFlags=0
+PartitionReadOnly=1</programlisting></para>
+
+      <para>The above defines the update mechanism for the Verity partition of the root file system. Verity
+      partition images are downloaded from
+      <literal>https://download.example.com/foobarOS_$v_$u.verity.xz</literal> and written to a suitable
+      local partition, which is marked read-only. Under the assumption this update is run from the image
+      itself the current image version (i.e. <literal>%A</literal>) is marked as protected, to ensure it is
+      not corrupted while booted. Note that the partition UUID for the target partition is encoded in the
+      source file name. Fixating the partition UUID can be useful to ensure that <literal>roothash=</literal>
+      on the kernel command line is sufficient to pinpoint both the Verity and root file system partition,
+      and also encode the Verity root level hash (under the assumption the UUID in the file names match their
+      top-level hash, the way
+      <citerefentry><refentrytitle>systemd-gpt-auto-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+      suggests).</para>
+
+      <para><programlisting># /usr/lib/sysupdate.d/60-root.conf
+[Transfer]
+ProtectVersion=%A
+
+[Source]
+Type=http-file
+Path=https://download.example.com/
+MatchPattern=foobarOS_$v_$u.root.xz
```

Comment:
Hmm, an snafu with this format is that the shell parses this as `foobarOS_${v_}${u}.root.xz`, because `_` is valid in a variable name. It seems that this difference could confused people.

Reply:
what about using `@`? That is often used when "templating" something

Reply:
sure, why not. will rework.